app/lib/oidc_client/account_manager.rb

```diff
@@ -8,6 +8,27 @@ def auth_uri(auth_request, mfa: false)
     )
   end
 
+  def callback(auth_request, code)
+    client.authorization_code = code
+
+    tokens = time_and_return "tokens" do
+      tokens!(oidc_nonce: auth_request.oidc_nonce)
+    end
+
+    response = get_ephemeral_state(
+      access_token: tokens[:access_token],
+      refresh_token: tokens[:refresh_token],
+    )
+
+    tokens.merge(
+      access_token: response.fetch(:access_token),
+      refresh_token: response.fetch(:refresh_token),
+      mfa: response.fetch(:result).fetch("level_of_authentication") == "level1",
+      ga_session_id: response.fetch(:result)["_ga"],
```

Comment:
Do these two also need a :
```
# TODO: remove `ga_client_id` and `cookie_consent` after we switch
# to DI, we're using a different approach there which doesn't go
# via the account-api
```
?

Reply:
No, because we'll just remove this class when the account manager goes away.